code-challenges.test.js

```diff
@@ -9,15 +10,46 @@
 
 // Reminder: The test will call your function
 // Run the file with the following command: $ yarn jest
+// Write a test for a function that logs "eat food" if you are hungry and "keep working" if you are not hungry
 
+// describe("areYouHungry", () => {
+//     it("returns eat food or keep working based on input", () => {
+//       expect(areYouHungry("yes")).toEqual("eat food")
+//       expect(areYouHungry("no")).toEqual("keep working")
+//     })
+// })
 
 // --------------------1) Create a function that takes in an array, removes the first item from the array and shuffles the remaining content.
 
 // a) Create a test with an expect statement using the variable provided. HINT: Check out this resource: https://jestjs.io/docs/expect#expectarraycontainingarray
 
-var colors1 = ["purple", "blue", "green", "yellow", "pink"]
+describe('colorArray', ()=> {
+    colors1 = ["purple", "blue", "green", "yellow", "pink"]
+    output1 = ["yellow", "blue", "pink", "green"]
+    colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
+    output2 = ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
+    it ('takes in an array, removes the first item from the array and shuffles the remaining content', ()=> {
+        expect(colorArray(colors1)).toEqual(expect.arrayContaining(output1))
+        expect(colorArray(colors2)).toEqual(expect.arrayContaining(output2))
+    })
+})
+// // my test needs a expect.any, .toContain , expect.arrayContaining(array)
+// // my function needs to take in an array of strings
+//     //colorArray()=>
+//     // function needs to remove first item from array.... unshift or shift to remove firs item array.
+//     // after first item is removed the strings in the array need to be shuffled
+//     // once shuffled I need to revist my test to see if it passed
+    
+const colorArray = (array) =>{
+    let scrambledArray = array.slice(1).sort(() => Math.random()-0.5)
+    //I looked up the Math.random()-0.5), I still don't knwo why the -.05 works.
+    return scrambledArray
+}
+
```

Comment:
Looks great. You could even make this a one-liner by returning the line of logic rather than creating a variable. `Math.random()-0.5` is a pretty cool trick. Play around with it a bit.